modules/Administration/Search/ajaxSubmit.js

```diff
@@ -50,13 +50,33 @@ $("form").submit(function (e) {
         return SUGAR.language.get("Administration", label);
     }
 
+    function alertMsg(message) {
+        var alertMsg = document.createElement("p");
+        var classAttr = document.createAttribute("class");
+        classAttr.value = "error";
+
+        alertMsg.setAttributeNode(classAttr);
+
+        var alertContent = document.createTextNode(message);
+        alertMsg.appendChild(alertContent);
+
+        var pageContent = document.getElementById("pagecontent");
+        pageContent.insertBefore(alertMsg, pageContent.firstChild);
+    }
+
     $.ajax({
         url: action,
         type: method.toLowerCase(),
         data: data
-    }).success(function () {
-        alert(translate("LBL_AJAX_SUBMIT_SUCCESS"));
```

Comment:
Is this text still used? If text does not necessary anymore, please remove string definitions from language file also.